spec/controllers/api/v1/packages_inventories_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::PackagesInventoriesController, type: :controller do
+  let(:user) { create(:user_with_token, :with_multiple_roles_and_permissions,
+    roles_and_permissions: { 'Reviewer' => ['can_manage_packages', 'can_manage_orders']} )}
+  let!(:package) { create(:package) }
+  let!(:package1) { create(:package) }
+  let!(:packages_inventory) { create(:packages_inventory, :gain, package_id: package.id) }
+  let!(:packages_inventory2) { create(:packages_inventory, :gain, package_id: package.id) }
+  let(:parsed_body) { JSON.parse(response.body) }
+
+  describe "GET packages_inventories" do
+
+    describe 'returns packages inventories for given package_id' do
```

Comment:
can you add a spec where you don't specify a package_id. We probably want an error message, doing an `index` on the inventory is risky so let's properly lock it down to a single package